src/chunk.c

```diff
@@ -3729,23 +3720,16 @@ show_chunks_return_srf(FunctionCallInfo fcinfo)
 		SRF_RETURN_DONE(funcctx);
 }
 
-static void
-ts_chunk_drop_internal(const Chunk *chunk, DropBehavior behavior, int32 log_level,
-					   bool preserve_catalog_row)
+void
+ts_chunk_drop_by_relid(Oid relid, DropBehavior behavior, int32 log_level, bool preserve_catalog_row)
 {
 	ObjectAddress objaddr = {
 		.classId = RelationRelationId,
-		.objectId = chunk->table_id,
+		.objectId = relid,
```

Comment:
Changes in this file look like unrelated minor refactoring, right? Let's merge separately.